doc/source/new/fqllogging.rst

```diff
@@ -171,6 +171,25 @@ For example: ``max_archive_retries: 10``
 
 FQL can also be configured using ``nodetool`` when enabling the feature, and will override any values set in the `cassandra.yaml` file, as discussed in the next section.
 
+Querying the state of FQL
+---------------------
+
+In order to know what state FQL is in, you may use nodetool command ``getfullquerylog``. It will print out whether FQL is enabled
+and with what configuration options, for example, like follows. If you reset or stop FQL, the configuration displayed will be taken from
```

Comment:
feel it reads better if `, for example, like follows. If` gets replaced with `; if`